lib/rules/no-tabs.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * @fileoverview Rule to check for tabs inside a file
+ * @author Gyandeep Singh
+ */
+
+"use strict";
```

Comment:
ah i see. I was focusing on use script, as i thought it needs to be moved some where.